src/scripts/main.js

```diff
@@ -1 +1,16 @@
 'use strict';
+
+const largeImg = document.getElementById('largeImg');
+const thumbs = document.getElementById('thumbs');
+
+thumbs.addEventListener('click', (ev) => {
+  ev.preventDefault();
+
+  const target = ev.target;
+
+  if (target.tagName === 'IMG') {
+    const link = target.closest('a');
+
+    largeImg.src = link.href;
```

Comment:
Accessing `link.href` assumes that `link` is not null. If `target.closest('a')` does not find an anchor element, `link` will be null, and accessing `link.href` will throw an error. Consider adding a null check for `link` before accessing `link.href` to prevent potential runtime errors.